public/app/screens/main/navigation/navigation.jsx

```diff
@@ -19,7 +19,7 @@ var Navigation = React.createClass({
 
       return (
         <Link to={item.linkTo} key={i} className={classNames}>
-          <div className={'cc_navigation__link-image ' + item.image} />
+          <i className={'cc_navigation__link-image ' + item.image}></i>
```

Comment:
See foundation docs for icon-bar with fonts.
Those svg images will be transformed into an svg font in the next pull request ;)